JH5/task10/src/homework/HtmlCreator.java

```diff
@@ -0,0 +1,99 @@
+package homework;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class HtnlCreator creates html code and returns string of html code
+ */
+public class HtmlCreator {
+
+    /**
+     * Colors of html-table
+     */
+    private static final String HEAD_COLOR = "#999999";
+    private static final String COL1_COLOR = "#efefef";
+    private static final String COL2_COLOR = "#D9D9D9";
+    private static final String COL_MAX_COLOR = "FF0000";
+    private static final String BORDER_COLOR = "#ffffff";
+
+    /**
+     * String HEADER contains header of html code
+     */
+    private static final String HEADER = "<html>\n " +
+            "<head>\n " +
+            "<title>Table</title>\n " +
+            "<meta charset = \"utf-8\">\n " +
+            "</head>\n " +
+            "<body>\n" +
+            "<table border=1 bordercolor =\"" + BORDER_COLOR + "\"CELLSPACING = 0" +
+            "cellpadding=\"1\"  width=\"630\" height=\"230\" align = \"center\">\n " +
+            "<tr bgcolor=\"" + HEAD_COLOR + "\"align = center valign = center>\n " +
+            "<th width=\"630\">Server</th>\n " +
+            "<th width=\"630\">Response, ms</th>\n";
+
+    /**
+     * String FOOTER contains finish part of html code
+     */
+    private static final String FOOTER = "</table>\n </body>\n </html>\n";
+
+    /**
+     * Body of the html code
+     */
+    private String HTML = "";
+
+    /**
+     * Method getHtml returns full html code of html-table
+     */
+    public String getHtml() {
+        return HTML;
+    }
+
+    /**
+     * Constructor HtmlCreator gets collection of ip addresses and creates
+     * a full html code of html table.
+     *
+     * @param addresses collection of ip addresses
+     */
+    public HtmlCreator (HashMap<String, Integer> addresses) {
```

Comment:
I'd say it's better to add server one-by-one to avoid case when developer has to create HashMap to generate table only for one server